lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -93,7 +93,7 @@ class FeatureCard extends React.Component {
 
   componentWillUnmount() {
     if (this.requests) {
```

Comment:
technically not necessary! left over from before when it wasn't defined up there already